po/de.po

```diff
@@ -242,11 +290,11 @@ msgstr "Einfügen _vor Auswahl"
 
 #: data/menu.ui.h:19
 msgid "Paste Interleave _Odd"
-msgstr "Verzahnt einfügen _ungerade"
+msgstr "Leere _ungerade Seiten einfügen"
 
 #: data/menu.ui.h:20
 msgid "Paste Interleave _Even"
-msgstr "Verzahnt einfügen _gerade"
+msgstr "Leere _gerade Seiten einfügen"
```

Comment:
Verzahnt is not great but with Leere I wouldn't have clue what this is doing.

Reply:
As far as I understand it, it's an option to include even or odd empty pages.

Reply:
No, I'm not aware of a function to add empty pages at all. This is about pasting the pages from clipboard in every second place. This is useful for example if you scanned all the fronts of your pages first and than all the backs and you want to put it back in a PDF in a way where each backside comes after it's front side.

Pages in Clipboard: A B C D
Pages displayed: 1 2 3 4
After paste interleave odd: 1 A 2 B 3 C 4 D

Reply:
Sorry, I misinterpreted it then. Thank you for explaining.
How about **Verschachtelt** instead of Verzahnt?

Reply:
Two more alternatives:
**Alternierend** or **Abwechselnd**?

Reply:
I think Verzahnt is better than Verschachtelt.
I'm okay with Alternierend, Abwechselnd or Verzahnt, not sure what is best.

Reply:
In that case I'd propose **Abwechselnd**.
But if you want, you may decide ;-)